### DISCUSSION TITLE
First OSS Contribution !

### DISCUSSION THREAD

**Discussion by simosathan9** [jMetal/jMetal#466] [Q&A]
Hey, I am very interested in this project and I was wondering if there is an issue I can help with as a first-time contributor. A little guidance would be very much appreciated. Thank you!

**Comment by ajnebro** [comment]
Hi.
  The contents of the jMetal project is the result of our research work and the contributions of other users. 
  
  If you are interested in multi-objective optimization with metaheuristics, depending on that then you have many choices. Feel free to contact me if you have any question.
  
  
  
  Antonio
